### ISSUE TITLE
Silo serial amr nesting bug

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1510]
Brian Ryujin reported a bug w/ displaying a Silo amr dataset.

Data is @ /g/g24/cyrush/visit/issue00737/ on OCF.

Create a plot of levels & turn off level 1.

In parallel we get the proper nesting behavior - but in serial zones are missing. 

Attached are images showing the issue. 

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 06/02/2011 02:25 pm
Original update: 10/13/2011 05:52 pm
Ticket number: 737
